packages/gcm/lib/apk/config.yaml

```diff
@@ -0,0 +1,11 @@
+registration-class: org.domokit.gcm.RegistrationIntentService$MojoService
+jars:
```

Comment:
4 of the jars are from the android SDK. I realized I can just use the ones in developer's SDK (which can be referenced by the $androidSdk var). The other 2 (gcm_lib and interfaces_java) are built by flutter engine.